generators/server/templates/src/test/java/package/security/DomainUserDetailsServiceIT.java.ejs

```diff
@@ -69,7 +69,7 @@ import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 @ExtendWith(RedisTestContainerExtension.class)
 <%_ } _%>
 <%_ if (databaseType === 'sql') { _%>
-@Transactional
+@Transactio nal
```

Comment:
Typo?

Reply:
Yeah, don't know how that happened, must have been in the last moment before I pushed (a little rushed because I wanted to share my progress)... :frowning_face: